### DISCUSSION TITLE
Frappe ERPNext install error

### DISCUSSION THREAD

**Discussion by DeveloperRaju23** [Dokploy/dokploy#413] [Show and tell]
I am trying to deploy A Odoo alternative [Frappe ERPNext](https://github.com/frappe/erpnext) but I got an error during the deployment.

2024/09/03 15:03:07 [emerg] 8#8: host not found in upstream "backend:8000" in /etc/nginx/conf.d/frappe.conf:2 with the container is restarting status.

**Comment by Siumauricio** [comment]
Hi @DeveloperRaju23 do you have a docker compose file to test it?

**Reply by DeveloperRaju23** [reply]
Dear @Siumauricio 

I am afraid I deleted it. But I suggest making a template out of it like Odoo.  Frappe ERPNext is fully open source, unlike Odoo.

**Reply by DeveloperRaju23** [reply]
Although you can look into it [docker compose file](https://github.com/frappe/frappe_docker/blob/main/pwd.yml)

**Reply by shubashbiswas** [reply]
This docker compose [GitHub](https://github.com/DeveloperRaju23/Dokploy-ERPNext) working perfectly 🤞. But I am not able to use multi instance setup. Although multi instance is not in focus right now. If someone able to do that that's will be much appreciated. Because in my case the client have different small business. So instead of getting different VPS for each business it will be more economical to run multiple instances in one setup. Frappe ERPNext support multi instance setup also.

My issue with my installation is by following frappe guide is that 2nd site isn't able to get online. I am getting 404 or bad gateway.

My goal is default site should have different website.
Example: 
1. Restaurants site will be `resturant.com`
2. Clothing store: `clothing.com`
3. Events Management: `events.com`

In my case the only default site is running perfectly not other two

[Frappe guide](https://frappeframework.com/docs/user/en/bench/guides/setup-multitenancy)

**Reply by Siumauricio** [reply]
I'm aware that multi instances of templates are not working, I need to tweak all the templates in order to support multiples instances of the same template, I will try to work on that very soon

**Reply by abdussamadbello** [reply]
@Siumauricio Thanks for your great work ! Will you create an issue to track this ?

**Comment by Siumauricio** [comment]
Implemented here https://github.com/Dokploy/dokploy/pull/1214
